Added more formatting options and changed name (#463)
Codecademy/docs: content/markdown/concepts/emphasis/emphasis.md

---
Title: 'Emphasis'

> Description: 'Emphasis can be added by making text bold, italic, or strikethrough.'

Reviewer: ```suggestion
Description: 'Applies importance by making text bold, italic, and/or strikethrough.'
```

In the meta-`Description`, it should start with a present-tense verb.

> Description: 'Applies importance to text by making it bold, italic, and/or strikethrough.'

Reviewer 2: So this is a regular entry, not a Term entry. So the Description should still start with the noun and not the verb.

```suggestion
Description: 'Emphasis applies importance to text in Markdown by making it bold, italic, and/or strikethrough.'
```

Reviewer 2: that's all from my end!

> ```markdown

Reviewer: ```suggestion
## Italics

```markdown
```

This avacado egg sandwich is _very_ good.
```

## Strikethrough

To make a strikethrough text, use 2 tildes `~` before and after a piece of text.

```markdown
~~Strikethrough text~~
```

For example:

```markdown
I did ~~not~~ build a website.
```

> Different formatting options can be combined:

Reviewer: ```suggestion
## Emphasis Combinations

Different formatting options can be combined:
```
